docs/explanations/data_collection.rst

```diff
@@ -0,0 +1,86 @@
+What data does Canonical collect from Ubuntu Pro machines?
+**********************************************************
+
+For the purposes of delivering Ubuntu Pro services to customers in compliance
+with the terms of the Ubuntu Pro subscription, some system data is sent to
+Canonical servers. This data is sent via a couple different methods, depending
+on the service and the purpose of that particular data element.
+
+This document categorizes data collection by method of collection.
+
+APT Package Downloads
+=====================
+
+If you have any of the following services enabled, then the following data
+collection method will be in use whenever downloading packages for one of
+these services.
+
+- ``anbox-cloud``
+- ``cc-eal``
+- ``cis``
+- ``esm-apps``
+- ``esm-infra``
+- ``fips``
+- ``fips-preview``
+- ``fips-updates``
+- ``realtime-kernel``
+- ``ros``
+- ``ros-updates``
+- ``usg``
+
+Whenever you ``apt install`` a package from a pro service (or ``apt upgrade``
+to a version of a package from a pro service), ``apt`` will make a GET request
+to ``esm.ubuntu.com`` that includes the package name and version and HTTP
+basic auth credentials that are tied to the Ubuntu Pro Subscription.
+
+For example, installing the ``hello`` package from ``esm-apps`` will result in
+a request that looks something like this:
+
+.. code-block:: text
+   https://bearer:$resource_token@esm.ubuntu.com/apps/ubuntu/pool/main/h/hello/hello_2.10-2ubuntu4+esm1_amd64.deb
+
+This request is necessary to download the Pro update and it necessarily
+includes the following data.
+
+- ubuntu codename (e.g. "jammy")
+- package name (e.g. "hello")
+- package version (e.g. "2.10-2ubuntu4+esm1")
+- package architecture (e.g. "amd64")
+
+Because this request is necessarily authenticated and the authentication token
+is tied to a particular Ubuntu Pro subscription, this data is inherently tied
+to the Ubuntu Pro subscription that authenticated access to the package.
+
+Livepatch Downloads
+===================
+
+If you have ``livepatch`` enabled, then the following data is sent in order to
+download the correct kernel patches:
+
+- kernel version (e.g. "6.8.0-38.38-generic")
+- machine architecture (e.g. "amd64")
+
+Because this request is necessarily authenticated and the authentication token
+is tied to a particular Ubuntu Pro subscription, this data is inherently tied
+to the Ubuntu Pro subscription that authenticated access to the package.
+
+
+Machine Activity Checks
+=======================
+
+Regardless of which services you have enabled, if a machine is attached to an
```

Comment:
I would reorder this as well:
"If your machine is attached to an Ubuntu Pro subscription, the following data is collected and updated regularly (regardless of which services are enabled or disabled).

Reply:
All of the document only applies if the machine is attached (ignoring the caveat of data retention for now), so this sentence was supposed to emphasize the irrelevance of which services are enabled for this section. The other sections open with which services need to be enabled for the section to be relevant, so I wanted to do the same here.

I'm happy to change it, but want to check if you would edit your suggestion given that context?

Reply:
That subtlety passed me by completely lol, but it makes sense and I'm happy for you leave this as it is.